cluster/addons/dashboard/dashboard-rbac.yaml

```diff
@@ -0,0 +1,49 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  labels:
+    k8s-app: kubernetes-dashboard
+    addonmanager.kubernetes.io/mode: Reconcile
+  name: kubernetes-dashboard-minimal
+  namespace: kube-system
+rules:
+  # Allow Dashboard to create 'kubernetes-dashboard-key-holder' secret.
+- apiGroups: [""]
+  resources: ["secrets"]
+  verbs: ["create"]
+  # Allow Dashboard to create 'kubernetes-dashboard-settings' config map.
```

Comment:
same question about the config map... create via manifest, let the dashboard get/update it

Reply:
Addon manager with `EnsureExists` could solve the problem, but not all of the users use it to deploy Dashboard. Dashboard 1.8 has its own method to recreate settings config map (thus this rule) to make it work not only with addon manager. I agree, that for next versions we should try to remove this rule.

Reply:
if they're using this set of manifests, including the secret and configmap will work regardless of whether they're using the addon manager, right?

Reply:
Yes, until they won't remove config map.

Done.